u3-views/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const {index} = require('./features/indexController')
+const { courseDetails,courseUnits,courseStudents } = require('./features/courseController')
+const { videoDetails1,videoDetails2 } = require('./features/videoController')
+
+const app = express()
+
+// Templates
+app.set('views', './views')
+app.set('view engine', 'hbs')
+app.engine('hbs', expHbs({
+  extname: '.hbs',
+  defaultLayout: false,
+  partialsDir: ['./views/partials', './views/layouts']
+}))
+
+
+// Middleware
+app.use(express.static('public'))
+
+// Routes
+app.get('/', index)
+app.get('/courses/:code', courseDetails)
+app.get('/courses/:code/units', courseUnits)
+app.get('/courses/:code/students', courseStudents)
+app.get('/videos/128', videoDetails1)
+app.get('/videos' , videoDetails2)
+
+// TODO later
```

Comment:
คุณทำสิ่งนี้แล้ว! ไม่ใช่ "สิ่งที่ต้องทำ" อีกต่อไป คุณสามารถลบบรรทัดนี้
You did this already! It's no longer "TODO". You can delete this line.